pkg/crc/segment/segment.go

```diff
@@ -79,9 +81,9 @@ func (c *Client) Upload(ctx context.Context, action string, duration time.Durati
 	}
 
 	properties = properties.Set("version", version.GetCRCVersion()).
-		Set("success", err == nil).
+		Set("success", err == nil || errors.Is(err, crcErr.VMNotExist)).
```

Comment:
Maybe extract a function like:

```
sentError := filter(err)

func filterError(err error) error {
  if errors.Is(err, crcErr.VMNotExist) {
    return nil
  }
  return err
}
```

Reply:
@guillaumerose sure that can be a helper function.